src/components/ProjetoCard.tsx

```
@@ -44,19 +44,22 @@ export function ProjetoCard({
   }, [projeto.imagemUrl, defaultImage])
 
   return (
-    <Card className="flex flex-col w-[510px] h-[215px] p-4 rounded-xl shadow-md shadow-black/30 bg-white text-black">
+   <Card className="flex flex-col w-full max-w-[510px] h-auto p-4 rounded-xl shadow-md shadow-black/30 bg-white text-black">
```

ESLint

[error] Insert `·`

(prettier/prettier)